docs/synthetics/http-monitor.md

```diff
@@ -74,6 +74,63 @@ By default, the UI adds the below conditions while creating an HTTP monitor. You
 * **Response Code** *equals* `200`
 * **Response Time** metric *less than* **20000 ms**
 
+
+### Configuring Custom Alerts based on JSON response
+In addition to our preconfigured alert conditions, you can also create your own based on the JSON returned in the response body. As mentioned above, this can be done using the **Response Body JSON** type, where [JSONPath](https://github.com/json-path/JsonPath) can be used to extract a specific value and compare it against the expected value.
+
+Let's say that you have an API which returns information about the internal status of your services. When everything is working fine, this is the response you get:
+
+```json
+{
+  "results":[
+    {
+      "name":"Database",
+      "status":"OK"
+    },
+    {
+      "name":"Redis",
+      "status":"OK"
+    },
+    {
+      "name":"Elasticsearch",
+      "status":"OK"
+    }
+  ],
+  "serviceStatusCode":"0"
+}
+```
+
+In the event where one of the services encounters issues, its status will change from `OK` to `Error`, like so:
+
+```json
+{
+  "results":[
+    {
+      "name":"Database",
+      "status":"OK"
+    },
+    {
+      "name":"Redis",
+      "status":"Error"
+    },
+    {
+      "name":"Elasticsearch",
+      "status":"OK"
+    }
+  ],
+  "serviceStatusCode":"2"
+}
+```
+
+With this JSON structure in mind, you can define a **Response Body JSON** alert condition like in the image below.
+
+![Custom Alert Condition](../images/synthetics/custom-alert-condition.png)
+
+The field is defined as `$.results[:].status`, meaning that all the `status` fields inside of `results` will be checked, and the monitor run will fail if any of them are equal to `Error`. Here's an example of a run failing upon one of these services reporting an error.
```

Comment:
I am not be seeing this in context in this PR, but it's a little confusing to go from talking about custom *metrics* (which I imagine as numerical values) to an example about looking for strings like `Error` in a response.  Ah, no, no, you didn't say "metrics" specifically above.  Ignore this.  But it will be interesting to see if we document how to  extract and what to do with both numerical values and string values.

Reply:
The approach should be identical as people will use JSONPath in the same fashion for both strings and numerical values